docs/content/gameobject/readers-writers.md

```diff
@@ -3,44 +3,48 @@
 [//]: # (TODO - See if `option` struct needs defining)
 
 # (GameObject-MonoBehaviour) Readers and Writers
-_This document relates to the [GameObject-MonoBehaviour workflow](../intro-workflows-spos-entities.md#spatialos-entities)._
+_This document relates to the [GameObject-MonoBehaviour workflow]({{urlRoot}}/content/intro-workflows-spos-entities#spatialos-entities)._
 
 Before reading this document, make sure you are familiar with:
-  * [Linking SpatialOS entities with GameObjects](./linking-spos-entities-gameobjects.md)
-  * [Workers](../workers/workers-in-the-gdk.md)
-  * (Optional) [SpatialOS schema](../glossary.md#schema).
-  * (Optional) [Read and write access](../glossary.md#authority)
+
+  * [Linking SpatialOS entities with GameObjects]({{urlRoot}}/content/gameobject/linking-spos-entities-gameobjects)
+  * [Workers]({{urlRoot}}/content/workers/workers-in-the-gdk)
+  * (Optional) [SpatialOS schema]({{urlRoot}}/content/glossary#schema).
+  * (Optional) [Read and write access]({{urlRoot}}/content/glossary#authority)
 
 Readers and Writers allow you to inspect and change the state of SpatialOS components using MonoBehaviours by letting you perform the following actions:
 
 **Reader**
-- Read the data of a SpatialOS component.
-- Read the authority state of your worker over a SpatialOS component.
-- Register callbacks for reacting to property value changed of a SpatialOS component.
-- Register callbacks for reacting to events corresponding to a SpatialOS component.
-- Register callbacks for reacting to changes to the authority state of your worker over a SpatialOS component.
+  * Read the data of a SpatialOS component.
```

Comment:
Whitespace around lists! (on the Writer too!)

Reply:
Fixed.